src/circles.rs

```diff
@@ -1,121 +1,131 @@
+#[cfg(feature = "vec")]
 use svg::node::element;
-use hex;
 
+#[cfg(feature = "pix")]
 const BACKGROUND_COLOR: [u8; 4] = [255, 255, 255, 0];
-pub const FOREGROUND_COLOR: [u8; 4] = [238, 238, 238, 255];
+pub(crate) const FOREGROUND_COLOR: [u8; 4] = [238, 238, 238, 255];
 
-/// Struct to store information about the circle
+/// Information about the circle
+#[cfg(feature = "pix")]
 struct Circle {
-    x_center: i32,
-    y_center: i32,
-    radius: i32,
+    x_center: f32,
+    y_center: f32,
+    radius: f32,
```

Comment:
Does the rendered circle entail a user API change? As in are these use for any visual verification purposes? If so, this should be a breaking change.